scripts/update-tree-sitter-bash-wasm.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Just for good practice, you can add -e to stop on error to both scripts

Reply:
Shellcheck complains on `#!/usr/bin/env bash -e` (https://www.shellcheck.net/wiki/SC2096), so I did `set -e` instead